src/Microsoft.AspNetCore.Sockets.Client/WebSocketsTransport.cs

```diff
@@ -15,7 +16,15 @@ public class WebSocketsTransport : ITransport
         private ClientWebSocket _webSocket = new ClientWebSocket();
         private IChannelConnection<Message> _application;
         private CancellationToken _cancellationToken = new CancellationToken();
+        private readonly ILogger _logger;
 
+        public WebSocketsTransport()
```

Comment:
Call the other ctor - if we add more initialization code it we will not have to do this in two places (also see my comment about loggerfactory below)